frameworks.mjs

```diff
@@ -167,11 +167,33 @@ export default [
     mainPackageName: "vue@^2",
   },
   {
-    id: "ember",
-    title: "Ember",
+    id: "emberPolaris",
+    title: "Ember (Polaris - preview)",
     img: "framework/ember.svg",
     eslint: {
-      files: ["**/ember/**"],
+      files: ["**/emberPolaris/**"],
+      plugins: ["ember"],
+      parser: "ember-eslint-parser",
+      extends: [
+        "eslint:recommended",
+        "plugin:ember/recommended",
+        "plugin:ember/recommended-gjs",
+      ],
+    },
+    playgroundURL: "https://ember-twiddle.com",
```

Comment:
Instead of twiddle (which idk if folks use), we have:
- repl: https://limber.glimdown.com
- stackblitz: https://stackblitz.com/github/ember-cli/editor-output/tree/stackblitz-app-output for full framework stuff

Reply:
Do you have a favorite as of now? limber.glimdown.com?

Reply:
I'd vote for https://new.emberjs.com/

Reply:
Has to be http tho. Https redirect doesn't work